tests/test_codecs.py

```diff
@@ -5,12 +5,38 @@
 import unittest
 from frontera.contrib.backends.remote.codecs.json import (Encoder as JsonEncoder,
                                                           Decoder as JsonDecoder,
-                                                          _convert, _reconvert)
+                                                          _encode_recursively, _decode_recursively)
 from frontera.contrib.backends.remote.codecs.msgpack import Encoder as MsgPackEncoder, Decoder as MsgPackDecoder
 from frontera.core.models import Request, Response
 import pytest
 
 
+def _compare_dicts(dict1, dict2):
```

Comment:
For me it was not immediately obvious, but I think the main difference from built-in dict comparison is this line https://github.com/scrapinghub/frontera/pull/233/files#diff-252cc4202700183e8f0c782fb28dbfc2R34, so it will catch e.g. python 2 dicts with equal str and unicode values, int/float, etc.